### PR TITLE
Fix error in list_group url and related method

### PR DESCRIPTION
**Pull request by thenaterhood**
Resolves an issue with URLs not matching the data string passed and an issue with callback not being retrieved if it happens to be None (it now defaults to None if it can't be pulled out of the dict, which should be alright since it appears other functions handle that function returning None).

### CODE REVIEW

File: rider/views.py [hunk 1]
```diff
@@ -133,9 +133,9 @@ def check_code_view(request, aff_id):
 
 def get_callback_from_request(request):
     if 'GET' == request.method:
-        return request.GET[u'callback']
+        return request.GET.get('callback', None)
```
Comment (bbesmanoff) [inline rider/views.py]: :+1: Does it pay to also fix [this one](https://github.com/TourTrak/devcycle-server/blob/staging/location_update/views.py#L292)?

Comment (thenaterhood) [inline rider/views.py]: Good catch, yeah may as well while I'm at it.

Comment (thenaterhood) [inline rider/views.py]: Fixed there too

**Comment by bbesmanoff** [comment]
:panda_face: :100:

**Comment by cman131** [comment]
:european_post_office: good with me :) :+1: merge away @thenaterhood

**Pull request merged**